internal/provider/mcc_schema_1_5.go

```diff
@@ -26,7 +25,7 @@ const (
 	HostRoleMSR = "msr"
 )
 
-func launchpadSchema14() schema.Schema {
+func launchpadSchema15() schema.Schema { //nolint:maintidx
```

Comment:
The linter things that this function goes too deep. It does go deep, but it is much more readable in this form than it would be to break it out into sub-functions that reduce this depth. The value here is that the function very much maps to the structure of the data.